### PR TITLE
MP-78 enable customization of uber-jar classifier

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

File: documentation/ecosystem/maven-plugin.adoc [hunk 1]
```diff
@@ -97,6 +97,10 @@ GAVs to be copied under MICRO-INF/deploy folder.
 |Can contain a list of artifactItems, which defines the dependencies with their
 GAVs to be copied under MICRO-INF/lib folder.
 
+|`uberJarClassifier`
+|microbundle
+|Specifies the classifier to use for the generated uber-jar.
```
Comment (OndroMih) [inline documentation/ecosystem/maven-plugin.adoc]: Maybe make this explicit and write "Specifies the maven artifact classifier"?

Comment (OndroMih) [reply documentation/ecosystem/maven-plugin.adoc]: Also for the `start` and `stop` goals.

File: documentation/ecosystem/maven-plugin.adoc [hunk 2]
```diff
@@ -276,6 +284,16 @@ you need to execute the plugin as: `mvn toolchains:toolchain payara-micro:start`
 |Defines payara-micro artifact with its coordinates. This information is used to
 identify the process id of the running payara-micro.
 
+|`useUberJar`
+|false
+|Use created uber-jar name that resides in target folder to identify the process id
```
Comment (OndroMih) [inline documentation/ecosystem/maven-plugin.adoc]: Better grammar:

"Use the name of the created uber-jar that resides in the target folder to identify the process id of the running Payara Micro instance"

**Comment by MarkWareham** [comment]
Can merge @jGauravGupta ?

**Pull request merged**